src/main/java/com/softserve/demo/repository/UserRepository.java

```diff
@@ -13,6 +13,8 @@
 
     User findByUsername(String username);
 
+    Optional<User> findByActivationCode(String activationCode);
```

Comment:
I think it`s better add new method in the end of class

Reply:
Yeah, but maybe not in every case. I don't think it is big problem here because I have resolved all conflicts. But for future if it can be add to the end I will add it to the end :)